core/drivers/fabric-driver/readme.md

```diff
@@ -71,6 +73,17 @@ Can pass in a config file for the driver to be run with: `DRIVER_CONFIG` (ex: DR
 
 NOTE: When specifying ensure that they match the config that the relay is using.
 
+**For docker-compose:***
+
+* `DOCKER_IMAGE_NAME`, `DOCKER_TAG`, and `DOCKER_REGISTRY` needs to specified based on [weaver-fabric-driver](https://github.com/hyperledger-labs/weaver-dlt-interoperability/pkgs/container/weaver-fabric-driver) image.
+* `DOCKER_PORT` is the driver port to be exposed, and `EXTERNAL_NETWORK` is the docker network in which fabric-network is running. 
+
+## Push Fabric Driver image to Github container registry
+
+* Create a Personal Access Token with write, read, and delete packages access in github.
+* Run `docker login ghcr.io` and use your github username and personal access token as password.
+* Run `make push-image` to build and push the image to github registry.
+
```

Comment:
Shouldn't we add instructions to create a `.npmrc`?

Reply:
Added.